prow/plugins/heart/heart.go

```diff
@@ -57,7 +57,7 @@ func helpProvider(config *plugins.Configuration, _ []config.OrgRepo) (*pluginhel
 		},
 	})
 	if err != nil {
-		logrus.WithError(err).Warn("cannot generate comments for heart plugin")
+		logrus.WithError(err).Warnf("cannot generate comments for heart plugin")
```

Comment:
Did you mean to user `%s` and `PluginName` here and in the files below?

Reply:
Indeed, my bad... I really should be more focussed.

Reply:
No worries.
/lgtm
/approve
/hold cancel